### ISSUE TITLE
Failure to load button styles

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/thanksroy#14]
I am not seeing any styles applying to the nav toggle buttons that open child list items:
<img width="293" alt="Screenshot of vertical nav menu with parent and child list items. A toggle button with no styles applied is selected" src="https://github.com/user-attachments/assets/25d0a246-ef9f-4594-aad0-621847bef05c" />
Also, something seems to have gone awry with the search submit button, which has some clipping. The target size is reduced to 30x6px (below screenshot includes focus indicator), which is below the minimum requirement of at least 24x24px. It appears the search font icon is still there, but the button height (and thus, fill) is reduced:
<img width="286" alt="Screenshot of search bar. Search submit button appears to be clipped inside its bounding box" src="https://github.com/user-attachments/assets/f76afe70-5198-4a06-a64a-0e55899bf86f" />

**Comment by aviannamiller** [comment]
Search submit button issue resolved by the changes in [#2280](https://github.com/omeka/omeka-s/issues/2280), though nav toggle button issue remains open.

**Comment by kimisgold** [comment]
I refactored the way the navigation works in https://github.com/omeka-s-themes/thanksroy/commit/3331ae6f6f02a7867266fee5dd5f7950caf12786. During the process, I think I've fixed the nav toggle button issue. Can you pull and see if it's resolved?

**Comment by aviannamiller** [comment]
Hmm, I am still seeing the same issue as the above image. What are the (currently empty) sub nav toggle buttons supposed to look like?

**Comment by kimisgold** [comment]
<img width="373" alt="Screenshot of Thanks Roy navigation, with sub navigation buttons represented as plus signs enclosed in circles." src="https://github.com/user-attachments/assets/7d83d433-17ab-473f-baef-867ed0207c83" />

This is what I'm seeing on the `develop` branch of Thanks Roy. I'm noticing that your version also has an older focus style, so I suspect there's something going on with either cached styles or a differing branch.

**Comment by kimisgold** [comment]
Okay, just pushed a fix. The subnav toggle inline style got caught up in the conditional for header background styles, but there's no real reason it should be there in the first place. I've just moved it back with the rest of the sass.

**Comment by aviannamiller** [comment]
All good on my end now. Thank you!
